react/react.d.ts

```diff
@@ -1180,6 +1180,11 @@ declare namespace __React {
         hyphens?: any;
 
         imeMode?: any;
+        
+        /**
+        * Aligns children in the main direction. For example, if children are flowing vertically, justifyContent controls how they align vertically. It works like justify-content in CSS (default: flex-start).
```

Comment:
this doc comment doesn't make sense outside of the context of React Native (it doesn't just *work* like `justify-content` in CSS, it literally *is* `justify-content`). can you use language from https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content instead?

style nits: `*` should be aligned, and line-length under 100 chars please